cmd/minikube/cmd/start.go

```diff
@@ -1763,6 +1752,48 @@ func validateDockerStorageDriver(drvName string) {
 	viper.Set(preload, false)
 }
 
+func validateSubnet(subnet string) error {
+	ip, cidr, err := netutil.ParseAddr(subnet)
+	if err != nil {
+		return err
+	}
+	if !ip.IsPrivate() {
+		return errors.Errorf("Sorry, %s is not a private IP", ip)
```

Comment:
The message should include something about this being a subnet, because there are many flags in minikube that are IP related `Sorry, the subnet provided is not...`